messaging/src/main/java/org/axonframework/eventhandling/pooled/PooledStreamingEventProcessor.java

```diff
@@ -498,16 +498,16 @@ public Builder spanFactory(@Nonnull EventProcessorSpanFactory spanFactory) {
         }
 
         /**
-         * Sets the {@link StreamableMessageSource} (e.g. the {@code EventStore}) which this {@link EventProcessor} will
+         * Sets the {@link StreamableEventSource} (e.g. the {@code EventStore}) which this {@link EventProcessor} will
          * track.
          *
-         * @param messageSource the {@link StreamableMessageSource} (e.g. the {@code EventStore}) which this
-         *                      {@link EventProcessor} will track
+         * @param eventSource the {@link StreamableEventSource} (e.g. the {@code EventStore}) which this
+         *                    {@link EventProcessor} will track
          * @return the current Builder instance, for fluent interfacing
          */
-        public Builder messageSource(@Nonnull StreamableMessageSource<TrackedEventMessage<?>> messageSource) {
-            assertNonNull(messageSource, "StreamableMessageSource may not be null");
-            this.messageSource = messageSource;
+        public Builder eventSource(@Nonnull StreamableEventSource<? extends EventMessage<?>> eventSource) {
```

Comment:
This method rename should also be included in the `api-changes.md`.

Reply:
Done.